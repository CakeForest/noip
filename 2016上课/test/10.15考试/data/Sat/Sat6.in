1000
37820249 2072136 20155624
91448890 56664940 33322740
23489943 65304220 62835745
25873472 78104992 55481948
29608580 65195650 16294386
40918420 36515805 22151724
26142640 37083427 22900306
18505873 86092944 20587169
51652480 45498576 97301260
15213744 26781000 75420855
21672708 81397482 6951550
65216396 54865114 68680308
9347560 35182304 95510555
83883450 51698310 65022488
23702876 527184 8505928
62486025 21774336 29083314
72517610 81093516 59393776
11203976 89943081 27097290
52054655 61361355 15136744
79053522 90946026 68120116
19118880 97481475 597126
44858256 98326597 79786401
45584896 3975793 34800
15959618 60474229 98274084
70519792 9534656 64684326
88487797 10115843 81053046
36703630 86436025 96241956
92446268 21611525 42955929
12429476 99654595 23078712
84966012 83563976 76702924
17724713 30264036 50839040
29012630 74240984 30898868
50698048 92912953 14619712
62811209 74887344 71514006
7529742 79195044 6131180
3946580 66580128 20248500
2078420 35586752 65271552
73677526 7892820 30115311
42854628 27598198 98306306
22777874 16828041 66814898
91869640 98448775 977022
66329406 95452263 18104926
73693220 62407025 9108702
89463323 57726620 27678880
84029436 96984864 88456880
34421700 36274572 74283890
81261704 3309952 12409738
75024460 86147382 61833998
91209962 20575368 26320731
66636336 83886680 92523860
38829864 69090888 68358552
83825543 59519816 34900096
60418380 1693029 85543892
5937120 75763000 10106858
13764414 30423612 86173442
1675520 94491488 2879717
35111664 55756039 52202680
16027192 7178330 32793472
66204676 64990200 91272364
81538662 60741320 29134272
40775037 41507351 54681560
78038660 59316057 50914392
14892280 53276400 47361585
74574990 94026365 93415283
30867522 59339252 87454696
94509891 21281500 77350634
7482748 99369816 11484002
54269984 15942834 18406320
20499747 14073930 63243152
56521102 473805 40215918
43519815 16201680 3867790
48363081 514490 171297
74282793 90042605 64672059
9837800 91816188 6072776
74499872 31465125 65354150
23416608 12739900 31643094
76141953 25494272 69848892
16677779 59534055 47684055
30972582 82383483 1762872
72571146 45796995 12090650
59086926 26443149 48249792
33475010 42905693 33074372
15233990 15563360 79331850
55753075 42501760 14740520
19768965 48974969 7987980
70650608 85389810 71281292
58891708 20587140 42275050
39324436 14047329 27227982
13909532 80861928 44087770
20075178 62066010 58910794
72720020 9861968 36375168
27340220 41982285 57046207
6043940 5223054 30371624
73047847 27343056 28896392
72381444 90719915 72550212
66085080 44709201 91820132
81458736 9297918 40760241
76225074 8527506 19183148
34518430 70089648 33125600
43133625 59017322 29647044
60109665 17725416 91120712
9606190 18076112 61078875
25887742 15279880 5868913
46410348 86652230 988800
63101546 35782608 41705130
55252050 39002251 11444888
23600817 69478000 49046936
38949382 23673336 48159612
97905616 32838063 98333660
64423490 56629937 45041158
36081898 39134448 8511924
64868474 60642730 3398043
61637208 61207628 42295944
63048609 65022902 26890280
57218930 7313107 36636374
37790080 1579968 6849816
671940 19679760 60971735
31086809 2166212 65747400
20085892 37106528 55948807
77151451 62185395 4335268
78192507 31021486 30614285
63398388 48680695 89069580
22462514 18584256 64313750
22156740 66868284 32396810
10651305 72716032 68119522
85437278 24258730 39909695
47710720 37847412 10939456
87113778 55027670 84103666
46625010 19104160 68419346
82507433 3024576 21396077
21411979 45082658 15307280
5588250 38688940 61042656
56489725 57639133 70675044
51060640 98126671 1892030
28534618 17290104 61144734
46719093 30813608 36391518
10450014 25398460 87842995
89370650 82691700 25263248
36453476 81839192 7668103
69954008 77442324 44001106
68052212 4394076 35869640
99963990 83011870 3473944
20821687 64591590 66058912
56517995 27182945 19587874
51204 18899890 8976678
87647104 49251813 93738402
1625420 60487456 40871730
8257600 3687750 43208875
66935454 59070594 19825084
33298894 51886112 68022176
51231088 9574448 85254625
79442984 38409481 67535904
8805888 3026344 17086910
97227436 15872750 61068220
17398680 62921472 6113851
1630670 48962558 4133796
95725044 11859179 18421920
75938174 9499930 46218700
72571600 42827594 69576155
6661960 86628392 41279781
68774577 45485866 74694612
31950776 62751180 35581195
13828320 68050908 60978830
81087000 25220992 33793905
68266766 26007264 94082005
53923838 83826310 42185111
6663646 85477470 21655533
36389040 39822260 30345222
98168856 96780827 65490965
69441680 69894668 5755212
44252110 95077334 39976622
33169226 60384979 39188360
32949826 43961356 80314852
53630583 85757475 23172095
64879241 83450337 12005980
526817 79469605 54693228
28272190 21801843 33269176
42331875 12611726 97144088
98180680 31934272 34493637
72558290 53387846 1552122
86796160 26262638 3254225
32833650 3998648 45769593
69565608 10347610 94712300
64904960 19142765 41180487
6219369 66493612 97832800
3224347 11987482 99154944
29271275 93113667 60034006
33439585 81638557 40778792
94274201 37653192 9205377
5099644 17777189 14643424
91777860 38919120 35368541
32147320 54707784 89605818
70361264 27254518 3437256
53713409 83496295 46529136
10833710 69041856 3383910
84053670 2864638 46216528
39794848 43184260 93267672
33563360 32401516 83341008
17671860 55695185 21132032
31439709 35748465 98432221
60380800 81610350 95532737
40786085 26127734 23626717
35314074 42676102 55848784
49101548 13434775 25964129
33708636 46277840 53760629
98609365 17978018 7572834
14646093 28513684 6902493
30346285 16531006 87340790
3306860 37539242 38884692
69757358 68299737 69112418
88121719 27434829 382170
46451735 62078567 94547255
21312080 9595026 74952338
34357287 23655895 26667930
82658964 9120828 90904542
70985750 44287710 6198812
29994125 57320196 78078933
82398188 54285560 50418085
78496194 50452620 92193920
32674000 92191176 51880040
16605084 85191566 9395172
54287636 32885536 27829982
4405526 39377157 13460006
65905480 18027792 61053200
20353899 67478661 71175182
18668210 76756944 11753936
49286100 99399000 61136965
19662084 98668091 55894116
28451968 14513127 93493340
79046280 80812450 12947615
10571112 52293856 45361965
97325318 76135332 96447216
27729248 1493850 12671342
77545430 84970171 51374647
83191880 93518053 598560
84360077 30609152 51755095
5885056 84003050 86388390
22058192 32933035 41563236
9419354 64864180 80113912
32729926 87854924 11387792
9699080 64421893 18030273
67277568 49048434 51828142
45410613 15186763 2207088
3471102 84605775 80872784
1523268 63017811 1818228
47298752 82846424 85877780
40731940 93724916 18218881
26854610 84466392 26702720
15136870 44860260 57545500
7090676 75585184 99948748
24523440 89261101 98673974
65868270 95516399 74588874
15822845 45596865 29359920
21593575 49549870 21908003
73615012 95762924 95304000
39031545 7885426 72313622
30397627 69288255 9200313
56927270 78215704 43590544
20514719 16003832 13071324
45567680 86406065 80635286
16285902 81923625 86800216
36624888 69598150 23798225
18606208 17548367 4658000
44065649 88814664 95656250
8271258 77186376 73180722
12162104 62188173 36525420
91931500 76270992 98320876
99249634 14383744 87575180
85368960 9664665 27650775
36589994 36625986 28208985
24210656 28748690 48818796
38618776 89383830 45565356
32797726 89682358 96316820
22447208 76795992 2208256
41922272 58361310 60599344
15325176 97763800 76201038
21415792 48341810 25550168
15084432 70921688 28438688
59164920 86044000 93777122
79600150 88453645 5181102
16451250 89894640 32339810
24073975 44877848 50309050
87917599 93322796 28600345
23386654 70263751 44599017
29576303 6433019 35716576
98597300 31909832 78359508
51074828 59480079 93479238
5194022 69555056 93177944
72951890 67024684 56499886
75899500 57028344 95149877
78412632 21984266 70104550
24796768 58790517 4795678
22631600 21498405 4918375
42749516 50442705 23039255
16211343 75068770 43626953
14086164 92783776 20558976
18934368 70152490 88263688
38465181 18377260 79337974
99903976 47389644 26093628
48653500 80901203 27739193
23226166 99459618 70976028
14626000 67826318 4729308
39132317 32387731 72998286
76730445 1519977 84554329
67056096 27225096 23139825
67446464 35722836 79651576
45152760 90861128 79221980
7866456 46404010 7866690
90169016 7121595 39179731
19298608 29958132 49576166
21600570 90879228 89605443
16901976 2450972 65573750
70451196 96683884 62136263
57327922 99167149 35197803
7932152 76230760 62484435
60487581 51107330 14637230
5030606 36022971 72464415
24596154 10196529 97931386
34168464 75109032 52817120
68000400 9086872 86951086
79786822 42011424 75725749
51807000 61657580 33027540
29557924 5871985 26572282
40648445 39739270 34405262
38365938 68044298 93357002
25220080 62265258 33867495
56947462 13213732 35151810
24179025 52472458 83367664
27519938 43506984 3230084
33995100 93134624 22290844
89289425 54486956 78009888
8121203 70923920 22961066
43254464 87918425 91884640
68618774 83299912 77438821
7343296 72437056 57944456
44688768 58726690 13295768
32164438 16409800 82682276
12644102 45626765 57219279
29794653 71786846 10822302
3107538 11407053 47317350
33612600 15465804 2756592
18430949 10008900 90459000
13754970 95061822 44122595
18509137 12963490 41723489
60591752 34117578 53971705
70015168 54797916 98980576
99199580 1290564 68441899
15688722 13580930 76850627
38174898 52213272 23736803
59505676 22385330 10206262
55509288 25439486 5751084
26298041 34075396 15688110
82485545 69918915 3275862
97902184 52940885 2738240
17606952 1545200 23694482
3768820 15062600 16722000
35947276 91046610 30133701
65426816 45287800 40088275
2573700 26385660 28826032
34255102 45226188 95445760
40219650 18928000 89433976
33153750 67073796 56634841
16993738 48650623 13685448
53135782 20347030 17650169
68211190 29326800 67427852
50081818 41315080 44598110
37668374 68484709 73082712
99605478 19415058 24267680
83236939 9983616 4629132
89332905 1651518 15373928
55236655 1700820 30459443
65236824 97704065 60157568
28739942 35841590 90127326
52162540 1305903 10731534
94022870 29842849 84087246
62928280 19128639 40116200
95573168 41905415 13671493
26117573 66439300 10725196
45554732 60671366 25454664
21398372 18472030 23608870
7287417 37923588 29717650
46854686 15287454 36561486
13827392 98761640 82783040
80509040 80266200 24854076
63337216 97867454 25770518
89404800 28617405 54978928
20852163 67918949 54346059
67237635 49067269 46540436
54490422 78013950 95873440
40203240 1850338 3473168
98624575 67073088 57529256
26655020 57935391 23965314
82571948 8532045 89738401
8515908 60274924 17215932
60483250 47034681 10865940
12512687 90939645 17511120
17267840 40468275 92258656
33112320 66789270 59920990
10040731 60291962 32683439
11260286 49587395 20745355
57232280 14238278 19048140
58183846 21478174 89380237
4224288 47302858 63346268
52576708 21870540 38742502
16169392 65646867 19073404
4874360 58286718 4667556
94600591 94155731 13034818
65364647 56720146 55722176
81110781 24949680 33997890
65596530 46697191 36176094
11347951 36245500 23802952
54108420 74164000 12262992
46933744 7258995 1252684
34613900 6759708 89400183
89864880 57650898 80458166
6659548 81549720 6435712
97444682 48424500 5572629
38273040 25816818 83834144
35125470 14004880 56328082
39105180 36211500 83675370
61154433 49448182 32642442
83413126 96959333 65876490
75624260 47022610 8880626
55102054 51682702 92019361
31924816 66658062 32301324
86352250 3058725 28787802
69606185 93884525 96936608
22012422 311244 43135920
6974424 23965724 80141800
6074690 23655872 60516000
39416289 73963420 125911
5230184 50667504 1188525
3567960 46177903 91001640
68429055 21985748 78588128
54962630 82690520 3763261
56008000 17628496 46471359
48439388 52604640 70453286
98839469 47384024 86501376
82323625 32754062 70993220
27307311 82364453 55173670
43204400 93541600 43575512
72382602 10689840 56172640
51914752 79164738 37687128
64997857 39899442 2446316
26893592 62696040 6611570
1142950 59727448 84661920
54155671 27943988 59820050
85551632 24588330 7635626
43929152 10150553 77463697
38411820 1566516 45465246
15698600 97561810 84043100
72108136 16380180 3971000
99500911 83660688 91848520
59582440 63456024 12057080
77737968 76646353 9888208
87925518 95016934 20979608
33287850 96717488 75276544
17198112 23181795 21274752
78144624 33325120 39038180
86397049 5007570 10726176
82129150 44808492 53633797
12104368 93956178 75785272
97077117 2216786 48845395
19221894 12837024 20282025
92927670 51323664 30409296
60879610 56602582 48707120
23718976 49819740 60393894
3481836 52292750 13612048
46017426 84546989 62263757
1242000 65175184 42334675
1565944 20860308 39690960
18600346 5763074 78033496
14572692 60554232 87337869
93521248 33002904 30967680
82464256 50968375 576198
1388999 1243808 4052392
31211550 626048 65048094
13978308 19176418 39239533
73717056 418660 28509920
91619086 54968840 42376360
36235336 87362408 80461096
47223246 67847735 61998900
74277618 1200474 98775860
53861780 91721975 51844072
54832932 11247696 37046196
12853878 56965430 82907768
24492440 15583470 7226750
3233900 26493189 72782855
79519648 97647060 82306370
91270671 51704704 86968554
31946576 49302958 14165100
98015428 7782646 19645416
82190850 71252222 24008508
57548127 10248008 65487566
61075244 31114538 72582792
61446920 27489896 68602552
8195733 15031838 25495359
47396804 79193014 87796246
24360560 85451910 22057056
54019771 48919990 63399713
89481660 17638793 17321475
68964048 14934556 14980245
1343231 44112427 34870126
43044014 62304590 17779441
31082836 20168936 872992
57296793 23299352 53250500
87010907 56145020 3323504
35484900 9900868 16529902
26142128 13933617 40400043
93735520 3152362 21158388
95252604 46813958 56839783
50141000 27649705 72258230
22307972 8325280 79230948
8566876 69400791 32182868
59428807 36197944 24580017
5088097 96304873 57944185
72610225 13450240 24414640
49959254 81008784 67078448
84582230 44648899 71505230
93733460 60967649 50511563
8286380 67199166 7341911
89309500 68944818 59511512
33672064 37553346 68667062
94436060 68555666 62022046
72681556 52992332 44964946
88553960 5867232 9865375
62274000 14930799 12223121
39073286 44303704 81669480
37329888 16937074 51412596
57280940 37889040 44430380
23270336 1878156 46707520
56967552 94055264 58134214
38863296 33184836 33424236
73553976 40557264 79295574
82129806 8036697 23240048
96358500 32222180 1550359
95733611 86776700 22939140
51040564 13637290 19557208
20281600 37036967 1895880
59289489 75996652 14643105
37538650 9670835 44521852
11851856 55633278 25528120
15446746 23688693 21471552
9118995 15056640 57678460
34702762 4833112 15159336
32270857 37708200 34078832
78356404 29578500 15086922
31137344 75235805 58021568
95938478 19468376 1917878
32700408 7104940 51020000
68803698 184030 43130960
28692654 33162202 21995028
11576952 69408410 96615654
47840739 48765465 71544900
60118954 30431906 96565680
87626140 76176660 68627902
62924526 94357902 2479800
94445736 89050 10933335
67861685 1891205 21749333
83195384 52081250 21105313
21724807 23082830 6200294
21610424 36250380 67182192
50733030 44696797 503200
71636234 57260217 14751165
99851625 9584874 33837581
53716000 99226400 8941704
67005104 41699990 37729800
76098650 69166568 13432055
61940002 766381 54904347
44286714 80595324 38809485
26691065 41671446 27769826
9437405 14031289 82015494
9831022 51422503 93891920
19889560 8437374 7442340
77401220 91332000 1412655
12985474 74900340 43969024
18374282 91252080 27130234
10279095 52233356 3673584
55142904 24146439 39560492
36512738 44186444 179050
12994936 18339216 16116986
89887853 46143139 73405024
9829417 13848000 65624660
58220632 70662292 7033104
24000320 90251760 17155702
36505326 18194968 35535640
11169711 38059447 35961708
69752424 24611528 69157200
49204088 46681559 20257710
44826705 27296227 34446030
36389724 8065322 67832398
33885640 17468760 9652500
69463672 73933280 8396840
13652378 76290776 37044595
70902448 10308880 44840226
11774422 68393910 24366930
47573682 5710810 9582366
47583747 56541264 34696980
29310312 39121698 84184352
58146428 68607758 72605096
2192062 55742646 19083665
84604476 69926078 69448174
15667669 90734415 65270701
21094080 70152243 41272015
52166368 4884480 88501400
48062586 29809160 47944243
24527320 46255808 11943440
54497343 95310700 57110599
71088071 72606576 47293150
714040 24835132 17728222
60045360 6390550 23688080
38598348 74965104 87973312
18565299 51035520 20850179
86641090 76400943 87132444
44644554 25928892 58441240
82948680 30674807 5264872
43767632 13093920 63601395
70306235 66351894 29523679
86306542 42755246 25701775
37943854 4209708 21964000
911293 11554427 13939600
5375340 50837832 73991180
47418516 97039539 77646920
65157280 42178014 59309032
12594542 95648090 90513924
70209625 20814620 32542552
15988450 50266388 99545759
75008185 78257038 7730635
91092560 99278480 35298120
30063347 16713405 65732514
54721952 72094076 47830869
63455423 62367560 76555516
20230207 10583065 32205316
12219408 70978101 76774767
37324306 29394736 69303648
68306208 60086379 5811410
51221688 87917200 46869676
5651885 37561570 41084790
18612200 62145357 65776571
13420480 25690598 77518590
40399572 54841438 28144231
81744898 95165650 56508333
9862235 42077840 72558976
65956992 12572925 50236615
15392156 6574336 19671652
33016000 55186072 39904434
58546521 6469424 28990770
10854296 25618120 28209575
89275026 930728 68396140
9368320 72479680 14217493
53617390 52009775 37493528
1319781 31587874 51297224
81485715 53204600 5696912
79100240 51553320 71816704
43009632 2355848 72530964
82817645 67931116 78802592
32891316 69620010 90078690
6680174 45875054 4850098
27826350 84888208 30047820
75056910 18963021 35383392
1987985 35148496 22977199
90420690 92153519 88794572
10227634 61751200 37115456
90654702 81173010 93584092
80849348 72691200 62064460
18326016 42092560 91714550
91028122 69623523 41317944
12866468 42526952 40771460
82546954 33933307 27078450
35826314 81575148 60575284
69860816 30694848 37798812
15454192 5566196 59834978
91315936 62530304 59036904
32066207 13938571 86868684
47111625 66384666 50401728
30699727 12188232 11282940
33296835 69751408 32366848
40075405 48097070 68666714
75226435 88597270 68882496
94518208 75811410 65404976
41326408 61156650 64841818
18578160 64551397 85407234
39670034 77027280 23187480
35241223 49149624 12710370
78943410 87814224 88435164
99325088 53512407 85378230
42613081 32972940 18491328
36817902 96832374 24263920
12125914 46007100 1817808
49764323 82989898 21894400
7370144 28971297 15567005
71902767 9801523 82494004
4655348 36475976 23072036
1799724 46562395 2058308
24311144 42076412 85273550
54864248 13464266 69313600
29970122 57096330 73302148
94257656 65762380 77656460
32863678 15346221 44877564
11750100 46820760 72799900
967031 39424536 3933718
92632575 69764920 499200
15991152 4285984 12736951
20592528 57368231 19584136
84057031 42079424 35274528
73369889 29306884 97047718
51166780 13100252 78830185
29517721 16997060 13180221
6796390 40918810 88089571
50420529 99697505 5352650
56643830 32065337 93289558
79741694 12462140 21543446
341100 26238985 99697704
15364740 72899860 61565296
73765754 2540958 12557325
49082780 10179761 58737817
58262868 36070914 60890014
41305704 44874828 43996444
62122970 86721285 62420112
966518 10748143 31271616
30003721 9127040 3161956
82184150 79035040 40537168
32436726 95841940 11856418
30536066 16103936 95459688
98643036 6869728 91689975
51494625 82716368 89396304
4939052 16922432 3850000
26910659 61852598 19468032
71962162 72569676 34706630
53898640 13372160 38321240
62632510 28203308 12626144
32421771 49851190 47816400
81418784 19457224 57691714
26816742 3493480 18117584
2318020 17640688 38437680
20130048 70771232 21446860
65848346 89192712 18053117
48436740 11500230 59503725
40988015 10715829 46918317
61235680 59659790 5720164
61533476 45409482 6648360
73886568 99487071 49663120
27218464 84822950 97191359
25038384 55990512 2253914
31290336 66835934 28615688
24849540 42284652 15779260
70303543 15806999 62154676
31012200 67951700 73153283
30154272 40890480 78852396
71876450 41760879 58104334
32383020 22877212 16090092
17224434 23384612 46897300
50494514 2377196 91523342
97596932 92067200 16279420
56400757 20353889 16887160
49726000 19855012 42860836
42960083 95908280 28182972
5807445 62773974 66744015
36936900 40403000 30399880
58694625 45519272 29715902
7346261 13198032 75096290
48062516 17965632 59471104
44370910 65035772 88853040
32001686 52074470 72660834
95581416 98618866 37916256
31387500 72417846 9873560
58754056 37600244 78126942
54547648 33519683 10688836
41977130 52254972 58094367
77364348 74366878 4883008
49737999 28553648 64478016
62224179 29627624 44732918
96484372 60199340 42295422
6303988 51852284 7281482
49518344 9487522 38102823
50079176 15847026 52106803
75357238 13120371 74412744
29170875 139932 57368522
72674695 2864560 27429454
32444335 61497210 71566730
16228778 73099202 1787740
43352758 57558521 49490130
77436950 86964185 6122656
20551227 98434845 41935648
84096890 52740016 92308880
93802700 7700607 62567352
91058313 16459670 16415616
44072456 16809463 60201672
982893 95137434 75839120
7302240 58023350 25890190
33405745 33539964 49767589
38237388 57457788 4185116
59249064 70843680 23906452
42748776 49613078 92714565
52362888 15021370 73536160
56711105 90146021 59767894
53749361 80589592 46809945
54428231 65738261 89765505
49660875 4274320 25455686
27747685 76581340 37016609
48529424 62615271 60310047
14027904 9855965 87334709
38740464 56424982 17473912
89531810 59570792 85861080
10197360 3068928 13188880
65853468 8901390 63071640
88560784 71444970 99330568
99072245 52389215 58692708
31780880 65655120 32826625
61823700 6164120 93097583
14926880 66101512 18218826
39001078 71538802 61788760
11043519 55185936 32557700
14549983 95482128 59571186
16236718 7663688 3161580
3756032 85734188 20122424
14295489 93804245 50456199
13907938 5039224 65566464
25641278 63232845 41224364
4457184 69726592 2035350
99917370 7931664 98797460
33411634 50222645 6821427
2312322 757482 7799490
3303590 17168890 36358800
42180674 3775755 49830804
51192978 86702103 85828248
46414891 67156244 23902096
91691872 79050432 95355182
10888855 99729594 37397702
6195000 34064934 25259035
38537178 91809307 21051479
26963886 23264602 1467944
36202180 73012500 19589976
942912 10841883 18259951
62162646 5369304 6419480
24999756 56596204 91950224
5825291 52325537 3559628
9633690 45858984 23379410
94334523 51163750 64695572
5463936 27878136 39138552
53724662 31477940 30447300
27858105 44544788 28429051
6014940 76061045 82560211
96986958 23356782 81235515
50229924 897160 33580699
79236320 31134744 93931931
54684244 51971049 6503225
36133865 69488518 93769988
43733390 19732656 53349624
76126334 49941456 6214868
54980605 4937540 71037846
80786000 75920480 28787364
36389316 50620505 98311808
4571632 91436129 33105490
21490889 99086288 17658820
57249729 43907786 73363827
57895774 10553400 51935736
16287353 95494290 95672508
81769446 73736234 30829744
37645200 97788525 57320800
44405979 38960604 10289714
36579840 39869059 39494744
594935 63936441 19924077
85022218 15453730 96338380
58015776 12347900 7874272
6013826 90189168 40653443
84762104 36463011 1153256
36936169 67265120 8460395
20563729 61620983 2472960
25358935 4787262 66951552
7560886 2532300 24634055
54487843 82494720 80780646
68714128 20143835 39240596
24711524 90406008 9575432
61097672 38697424 64486558
32365038 62910239 18509137
81958656 6313104 32782896
26965288 71943810 91061330
85940814 13233104 44070528
21483262 23974202 73618200
1509384 77609547 19829696
41484972 34950945 20227200
12852614 25311405 28403283
4231394 85796376 30478765
56087262 92360750 83036754
24347884 71241858 62077473
87929854 61175377 29947408
12603248 46644662 291725
43161480 59733528 22349678
74710036 40017627 26977508
60165600 86612356 35449642
77186848 53697648 45407502
11600910 90894510 74272320
2380517 58380406 1199016
10301148 3021751 15261598
14993020 60092336 49770774
22767397 4567275 6068776
67949610 65484648 84948050
83782784 73390894 52182370
15236397 54133638 7360821
78390782 85360384 24113378
45461792 44836050 50809856
95109200 7797380 7510776
24434324 41758699 18489296
5844560 24228974 1467878
97627798 24767136 54000595
20065238 68795106 12488066
92321236 52009316 6159386
83694193 60823703 9931777
44314672 6950007 68256400
77599282 89149240 3612340
74235376 42221300 89292270
79918680 93790760 96540118
87097478 67827375 68559116
15150631 97691786 18478780
3467052 411334 22586898
13456474 74111560 38415341
63765685 4089345 6703060
7420125 58415543 6585362
4151442 29210082 94255694
66825077 8601588 67022198
61811552 84225178 22626698
38959592 78335860 44663165
33276480 46425368 58058880
12321328 77396870 98272040
48292159 4060640 28612760
16034117 91104244 70914672
65104648 74898582 46341800
86678220 42617869 51919994
26213064 25929998 65591444
34531345 28842678 31370608
92287613 32924766 97748616
29219100 66055801 27934355
12825605 25398708 55596698
93107626 29389930 5473142
72730700 40572000 53364662
14448640 410184 6546112
42486752 50676504 70352802
61955552 21651456 82272705
86092344 17144160 2704574
64302596 4970350 46775808
2641106 50529288 87019719
55877624 40106570 91883897
6211276 7640390 6105000
806400 13669804 69194250
38820246 43825239 24681920
19054415 69731172 98295845
78451610 16088262 82278192
24839800 19539241 60099001
34184120 93301646 23794290
49010691 78348676 40179840
71459968 69102000 40720540
13012670 12645632 10963874
65397340 86631820 94922235
53100552 4127394 36483092
41105568 81053125 71869980
68011215 42211210 46269076
26618675 10133192 67103290
18060710 7436287 41377680
29561200 64849937 6300390
33772861 64679233 16227796
58829824 79301072 64921384
2758020 67922117 7545248
48093346 66844395 60851287
40686948 14340842 9658350
7938126 98800167 57405808
69865776 3432986 84541625
11301488 9708300 42568798
79805574 65805900 28248815
59137263 56287646 57848374
89870627 15782224 90230580
45167120 64079960 11247660
77741837 48447092 72370496
64925000 39207920 81106152
95412140 20892458 42517150
39279120 23244138 9958795
65903450 49829542 60359344
1382210 5571746 51239750
98211158 72018834 3712590
78283520 60676913 5950245
21958588 10581462 96832320
42846986 89139224 38534760
6755184 25084310 40069051
60206270 55297656 14775989
68014355 72861740 53153230
70268771 85162328 79786944
97062808 57454225 1051200
14186775 23299544 90257854
12549216 51498872 60614128
61304624 79941880 56495140
7119702 62402214 77184236
46664480 89840000 57058920
5814120 29702560 40720316
45894452 32348384 45116416
88900515 53156036 2844304
77721226 70981416 52308614
73195920 37759000 17550897
20234214 46494976 26352384
7850712 53202734 99184519
83219328 15864840 33091353

