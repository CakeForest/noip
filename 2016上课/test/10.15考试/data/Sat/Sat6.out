2762385513480
